src/plays/product-entry-form/Readme.md

```diff
@@ -0,0 +1,27 @@
+# Product entry form
```

Comment:
@mohitdevelops Add some more details in the Readme considering some one would like to know how you had implemented this piece. More like a high level article.